src/transport/TransportMgrBase.cpp

```diff
@@ -1,6 +1,6 @@
 /*
  *
- *    Copyright (c) 2020 Project CHIP Authors
+ *    Copyright (c) 2021 Project CHIP Authors
```

Comment:
I think this needs changing to 2020-2021 based on what I see in other files (i.e. we put ranges instead of changing the year)